### PR TITLE
Add ifevals for Katello and Red Hat only content

### PR DESCRIPTION
**Pull request by melcorr**
As part of

Bug 1797957 - Write upstream/downstream conditions for Installing
Satellite Server (Connected) guide

https://bugzilla.redhat.com/show_bug.cgi?id=1797957

### CODE REVIEW

File: guides/common/modules/installing_overview.adoc [hunk 1]
```diff
@@ -2,12 +2,32 @@
...
 There are two methods of installing {ProjectServer}:
+endif::[]
+
+ifeval::["{Build}" == "foreman"]
+
+On CentOS, you can install {Project} with or without the Katello plug-in. If you are a new user, consider installing {Project} with the Katello plug-in.
+
+On Debian or Ubuntu operating systems, installations with the Katello plug-in are not supported.
+
+endif::[]
```
Comment (tahliar) [inline guides/common/modules/installing_overview.adoc]: This ifeval is duplicated.

File: guides/common/modules/synchronizing_the_satellite_tools_repository.adoc [hunk 2]
```diff
@@ -1,8 +1,15 @@
...
 = Synchronizing the {Project} Tools Repository
 
+ifeval::["{Build}" == "foreman"]
+If you use the Katello plug-in, you can synchronize the {Project} Tools repository from the Red Hat Content Delivery Network (CDN) to your {Project}.
+This repository repository provides the `katello-agent` and `puppet` packages for clients registered to {ProjectServer}.
+endif::[]
+
+ifeval::["{Build}" == "satellite"]
 Use this section to synchronize the {Project} Tools repository from the Red Hat Content Delivery Network (CDN) to your {Project}.
 This repository repository provides the `katello-agent` and `puppet` packages for clients registered to {ProjectServer}.
```
Comment (tahliar) [inline guides/common/modules/synchronizing_the_satellite_tools_repository.adoc]: Double "repository" in both ifevals, so I guess it was already there.

File: guides/common/modules/AWS_Deployment_Scenarios.adoc [hunk 3]
```diff
@@ -9,17 +9,21 @@ There are three deployment scenarios for {ProjectName} in Amazon Web Services:
 . Connecting on-premise and AWS region
 . Connecting different regions
 
+ifeval::["{Build}" == "foreman"]
+The graphics in this section are Red Hat illustrations. Non-Red Hat representations of these scenarios are welcome. If you want to contribute alternative illustrations, raise a pull request in the https://github.com/theforeman/foreman-documentation[Foreman Documentation] Github page.
```
Comment (tahliar) [inline guides/common/modules/AWS_Deployment_Scenarios.adoc]: s/Github/GitHub ;)

Comment (melcorr) [reply guides/common/modules/AWS_Deployment_Scenarios.adoc]: Martin would kill me :-P

**Pull request merged**